### DISCUSSION TITLE
We can use YouTube video on "Playlist URL"

### DISCUSSION THREAD

**Discussion by MrLabbrow** [ant-media/Ant-Media-Server#5989] [FAQs]
Hay, If I want to use the YouTube video URL on the "Playlist URL" section and start "Start Broadcast" the live stream will work.
When I add a YouTube URL to the "Playlist URL" section it saves the URL on the playlist, but my livestream crashes
![Screenshot 2023-12-31 015837](https://github.com/ant-media/Ant-Media-Server/assets/125576462/0b8266ad-da77-4ed9-a540-afcd22bf8968)
...

**Comment by USAMAWIZARD** [comment]
@MrLabbrow  it is not directly possible to add YouTube URL in the Playlist but you can use this way to stream  from a youtube video to AMS without uploading it to AMS

Streaming youtube video without uploading it to Ant Media Server 
you can also use ffmpeg youtube dl like this 
```
python3 -m pip install --force-reinstall https://github.com/yt-dlp/yt-dlp/archive/master.tar.gz

ffmpeg -re -stream_loop -10 -i  "$(yt-dlp  -g -f best https://www.youtube.com/watch?v=xvFZjo5PgG0)" -c:v libx264 -x264-params keyint=80:scenecut=0  -preset fast -tune zerolatency -f mpegts  srt://AMS_IP:4200?streamid=WebRTCAppEE/stream11

https://AMS_IP:5443/WebRTCAppEE/play.html?id=stream11&playOrder=hls
```
